spec/02-integration/04-admin_api/21-services_routes_spec.lua

```diff
@@ -97,6 +97,31 @@ for _, strategy in helpers.each_strategy() do
             assert.equals(60000, json.read_timeout)
           end
         end)
+
+        it_content_types("creates a service with url (https based)", function(content_type)
```

Comment:
Isn't the proper name for this test: `'port' defaults to 443 when 'url' scheme is https`?